schema/vulntology-json-schema-1.0-draft.json

```diff
@@ -298,6 +304,7 @@
                 "id",
                 "hasImpactMethod",
                 "affectsContext",
+                "hasEntityRole",
```

Comment:
This reverses a prior PR (usnistgov/vulntology#73) where we reduced required properties.